.bumpversion.cfg

```diff
@@ -1,15 +1,19 @@
 [bumpversion]
 commit = True
 tag = True
-current_version = 0.5.10
-parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\-(?P<release>[a-z]+))?
+current_version = 0.5.10-dev+2.3.4-meta
+parse = (?P<major>\d+).(?P<minor>\d+).(?P<patch>\d+)(-(?P<release>[^\+]+))?(\+(?P<metadata>.*))?
 serialize = 
-	{major}.{minor}.{patch}-{release}
+	{major}.{minor}.{patch}-{release}+{metadata}
+	{major}.{minor}.{patch}+{metadata}
 	{major}.{minor}.{patch}
 
-[bumpversion:file:setup.py]
+[bumpversion:part:metadata]
+keep_value = True
```

Comment:
I would call this `independent`, because the `metadata` part can still change (if you change it explicitly).

Reply:
hmm, I'm not sure if I would grok exactly what `independent` means at a first glance. Changing it explicitly is out-of-scope of bumpversion and in the context of bumpversion `keep_value: True` seems more straightforward.

Reply:
I have to agree with @c4urself as I as well wouldn't know how to interpret `independent`. But I am still open for other suggestions.

Reply:
Ok, now I see where this `independent` actually comes from (ADVbumpversion). If you plan to merge both forks again I would be find with `independent`.
Are there any plans to do that?